frontend/src/js/components/TemplateList.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+var TemplateList = React.createClass({
+    render() {
+        return (
+            <div className='left-side-list scroll-area-list-parent'>
+                <nav className="navbar navbar-default">
+                    <div className="container-fluid">
+
+                        <div className="">
+                            <ul className="nav navbar-nav navbar-right side-nav">
```

Comment:
This "add edit remove" menu is going to be used in a lot of places. That can probably get extracted out into a separate piece.